projects/1.20/assets/tinkers-jewelry/tinkersjewelry/lang/zh_cn.json

```
@@ -0,0 +1,262 @@
+  "modifier.tinkersjewelry.fly_gem": "飞行",
+  "description.tinkersjewelry.flight": "%s秒飞行",
+
+  "modifier.tinkersjewelry.armor_gem.description": "给予额外护甲值",
+  "modifier.tinkersjewelry.armor_gem.flavor": "给予额外护甲值",
+  "modifier.tinkersjewelry.armor_gem": "护甲",
+
+  "modifier.tinkersjewelry.piercing_gem.description": "无视目标的部分护甲值",
+  "modifier.tinkersjewelry.piercing_gem.flavor": "无视目标的部分护甲值",
+  "modifier.tinkersjewelry.piercing_gem": "穿甲",
+
+  "modifier.tinkersjewelry.speed_gem.description": "增加移动速度",
+  "modifier.tinkersjewelry.speed_gem.flavor": "增加移动速度",
+  "modifier.tinkersjewelry.speed_gem": "疾速",
+
+  "modifier.tinkersjewelry.fireresistance_gem.description": "给予抗火",
+  "modifier.tinkersjewelry.fireresistance_gem.flavor": "给予抗火",
+  "modifier.tinkersjewelry.fireresistance_gem": "抗火",
+
+  "modifier.tinkersjewelry.silk_gem.description": "给予精准采集",
+  "modifier.tinkersjewelry.silk_gem.flavor": "给予精准采集",
+  "modifier.tinkersjewelry.silk_gem": "精准采集",
+
+  "modifier.tinkersjewelry.fortune_gem.description": "给予时运",
+  "modifier.tinkersjewelry.fortune_gem.flavor": "给予时运",
+  "modifier.tinkersjewelry.fortune_gem": "时运",
+
+  "modifier.tinkersjewelry.waterbreathing_gem.description": "给予水下呼吸",
+  "modifier.tinkersjewelry.waterbreathing_gem.flabour": "给予水下呼吸",
+  "modifier.tinkersjewelry.waterbreathing_gem": "水肺",
+
+  "modifier.tinkersjewelry.nightvision_gem.description": "给予夜视",
+  "modifier.tinkersjewelry.nightvision_gem.flavor": "给予夜视",
+  "modifier.tinkersjewelry.nightvision_gem": "夜视",
+
+  "modifier.tinkersjewelry.poison_gem.description": "使得目标中毒",
+  "modifier.tinkersjewelry.poison_gem.flavor": "使得目标中毒",
+  "modifier.tinkersjewelry.poison_gem": "猛毒",
+
+  "modifier.tinkersjewelry.flame_gem.description": "点燃目标",
+  "modifier.tinkersjewelry.flame_gem.flavor": "点燃目标",
+  "modifier.tinkersjewelry.flame_gem": "烈火",
+
+  "modifier.tinkersjewelry.cleanse_gem.description": "移除负面效果",
+  "modifier.tinkersjewelry.cleanse_gem.flavor": "移除负面效果",
+  "modifier.tinkersjewelry.cleanse_gem": "净扫",
+
+  "modifier.tinkersjewelry.health_gem.description": "给予额外生命值",
+  "modifier.tinkersjewelry.health_gem.flavor": "给予额外生命值",
+  "modifier.tinkersjewelry.health_gem": "生命",
+
+  "modifier.tinkersjewelry.reach_gem.description": "给予额外实体交互距离",
+  "modifier.tinkersjewelry.reach_gem.flavor": "给予额外实体交互距离",
+  "modifier.tinkersjewelry.reach_gem": "远触",
+
+  "modifier.tinkersjewelry.trade_gem.description": "与村民交易时获得降价",
+  "modifier.tinkersjewelry.trade_gem.flavor": "与村民交易时获得降价",
+  "modifier.tinkersjewelry.trade_gem": "贸易",
+
+  "modifier.tinkersjewelry.exp_gem.description": "增加获得的经验",
+  "modifier.tinkersjewelry.exp_gem.flavor": "增加获得的经验",
+  "modifier.tinkersjewelry.exp_gem": "经验",
+
+  "modifier.tinkersjewelry.resistance_gem.description": "给予抗性提升效果",
+  "modifier.tinkersjewelry.resistance_gem.flavor": "给予抗性提升效果",
+  "modifier.tinkersjewelry.resistance_gem": "抗性",
+
+  "modifier.tinkersjewelry.luck_gem.description": "增加佩戴者的幸运",

```

感觉怪怪的

---

这个确实，主要原因是幸运本身是个attribute名
```suggestion
  "modifier.tinkersjewelry.luck_gem.description": "让佩戴者更加幸运",
```
改成这样